tests/atomistics/structure/test_analyse.py

```diff
@@ -144,6 +144,14 @@ def test_get_interstitials_fcc(self):
             msg='Distance variance in FCC must be 0'
         )
 
+    def test_strain(self):
+        bulk = StructureFactory().ase.bulk('Al', cubic=True)
+        epsilon = 0.1*(np.random.random((3,3))-0.5)
```

Comment:
```suggestion
        epsilon = 0.1*(np.random.random((3, 3))-0.5)
```

Please get a PEP8 linter you VIM(Emacs?)-nutcase 😝